internal_testsuites/golang/testsuite/starlark_exec_test/starlark_exec_test.go

```diff
@@ -40,7 +40,7 @@ def run(plan, args):
 		service_name = "test",
 		command = %v,
 	)
-	exec_result = plan.exec(exec_recipe)
+	exec_result = plan.exec(exec_recipe, acceptable_codes=[0, 1])
```

Comment:
is this non deterministic?

Reply:
We run this test over multiple scripts (it's parametrized) and do the assert later. It's also a nice way to test the feature